composer.json

```diff
@@ -56,7 +56,8 @@
   "require": {
     "clearfw/clearfw": "~1.2.0",
     "easyrdf/easyrdf": "^0.9.0",
-    "doctrine/dbal": "~2.5.0",
+    "doctrine/dbal": "^2.9.0",
+    "doctrine/annotations": "1.*",
```

Comment:
see https://packagist.org/packages/doctrine/annotations#v1.6.1
it implicitly bind you code to php 7.1 as a dep now :)

Reply:
So it cannot be merged I guess?